14_team6_agentdesign/index.tex

```diff
@@ -1 +1,422 @@
+\begin{figure}[htp]
+    \centering
+    \includegraphics[width=12cm]{14_team6_agentdesign/images/roles.png}
+    \caption{Distribution of Roles}
+    \label{fig:roles}
+\end{figure}
+
+
+\subsection{IIGO Payments} \label{subsec:Team6_Eval_IIGO}
+
+In Figure~\ref{fig:IIGO}, compared to other islands, we manage to run with a lower expected tax level and actually paid more than that. Moreover, we never take more resources than allocated by President and never break the rule, which is also one of our advantages, as we don’t have to waste our resources in sanction payments for breaking rules. By doing this, we reduce the risk of common pool resource exhaustion in a short time, which contributes to the stability and sustainability for the whole system.\\
+\begin{figure}[htp]
+    \centering
+    \includegraphics[width=12cm]{14_team6_agentdesign/images/IIGO cost.png}
+    \caption{IIGO Costs}
+    \label{fig:IIGO}
+\end{figure}
+
+\subsection{IITO visualisation} \label{subsec:Team6_Eval_IITO}
+The following plot shown in Figure~\ref{fig:IITO} visualises the transactions between islands in the IITO sessions. The size of a bubble represents the total magnitude of resources traded by each island. The width of each connecting edge between two islands represents the magnitude of transactions between the two islands. Islands that gave more resources than they receive are indicated by a red border, while islands that received more than they gave are indicated by a green border.
+
+From the plot we could conclude that resources we have received are more than that given by us. Also, our bubble is the penultimate small one, which indicates that we are somewhat inclined to be self-isolated and lack of trade exchanges with other islands. This also could indicate that the friendship with other islands is somewhat not getting along really good based on our own opinion formation towards the other islands.
+\begin{figure}[htp]
+    \centering
+    \includegraphics[width=8cm]{14_team6_agentdesign/images/IITO.png}
+    \caption{IITO Visualisation}
+    \label{fig:IITO}
+\end{figure}
+
+\subsection{Resources over time} \label{subsec:Team6_Eval_Resources}
+
+Line charts listed in Figure~\ref{fig:all teams} show the changes about resources on each island and the common pool, as well as the sum of all resources over time. From the simulation results, we could conclude that every island manages to survive long enough, which indicates that our agent adapts well to the operation of the entire system and could automatically take advantage of situations in different specific situations. 
+
+In addition, the resources change curve of our agent as seen in Figure~\ref{fig:team 6} is much smoother than that of some other agents, which shows that we actually have a good grasp of balancing out between income and expenditure of our resources. \\
+\begin{figure}[htp]
```

Comment:
you are advised to do
`\begin{figure}[H]` for all images

Reply:
thanks, will change

Reply:
changed